src/components/application_manager/include/application_manager/commands/command.h

```diff
@@ -113,7 +113,7 @@ class Command {
  */
   virtual void SetAllowedToTerminate(const bool allowed) = 0;
 
-  enum CommandOrigin { ORIGIN_SDL, ORIGIN_MOBILE };
+  enum CommandSource { SOURCE_SDL, SOURCE_MOBILE, SOURCE_HMI };
```

Comment:
what messages are from SOURCE_SDL?

Reply:
@yang1070 messages with SOURCE_SDL - messages that the SDL sends to the mobile (but not those that came from the HMI)
for example if we received too many requests : https://github.com/smartdevicelink/sdl_core/blob/feature/rc_refactoring/src/components/application_manager/src/rpc_service_impl.cc#L203